sdk/servicebus/service-bus/src/serializers/ruleResourceSerializer.ts

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { AtomXmlSerializer, HttpOperationResponse } from "@azure/core-http";
+import * as Constants from "../util/constants";
+import { serializeToAtomXmlRequest, deserializeAtomXmlResponse } from "../util/atomXmlHelper";
+
+/**
+ * Represents settable options on a rule
+ */
+export interface RuleOptions {
+  /**
+   * Name of the rule.
+   */
+  name?: string;
+
+  /**
+   * Defines the expression that the rule evaluates. The expression string is interpreted as a SQL92 expression which must evaluate to True or False. Only one between a correlation and a sql expression can be defined.
+   */
+  sqlExpressionFilter?: string;
+
+  /**
+   * Defines the expression that the rule evaluates. Only the messages whose CorrelationId match the CorrelationId set in the filter expression are allowed. Only one between a correlation and a sql expression can be defined.
+   */
+  correlationFilter?: string;
+
+  /**
+   * Defines the expression that the rule evaluates as a true filter.
+   */
+  trueFilter?: string;
+
+  /**
+   * Defines the expression that the rule evaluates as a false filter.
+   */
+  falseFilter?: string;
+
+  /**
+   * Defines the expression that the rule evaluates. If the rule is of type SQL, the expression string is interpreted as a SQL92 expression which must evaluate to True or False. If the rule is of type CorrelationFilterExpression then only the messages whose CorrelationId match the CorrelationId set in the filter expression are allowed.
+   */
+  sqlRuleAction?: string;
+}
+
+/**
+ * Represents all attributes of a rule entity
+ */
+export interface Rule extends RuleOptions {
```

Comment:
RuleOptions and Rule has same properties with different casing...
For example, `name`, `Name` etc.